src/envoy/mixer/http_filter.cc

```diff
@@ -96,7 +96,17 @@ class Config : public Logger::Loggable<Logger::Id::http> {
           __func__);
     }
 
-    http_control_ = std::make_shared<HttpControl>(mixer_server);
+    std::map<std::string, std::string> attributes;
+    if (config.hasObject("attributes")) {
+      for (const std::string& attr : config.getStringArray("attributes")) {
+        attributes[attr] = config.getString(attr);
+      }
+    } else {
+      log().warn("No attributes is specified in the config: {}", __func__);
```

Comment:
Open https://github.com/istio/proxy/issues/88 for first issue.  Did not quite understand the second issue yet.  Will open a separate issue once I understand it.